reflect/src/main/java/dagger/reflect/ReflectiveMembersInjector.java

```diff
@@ -87,43 +87,43 @@
       }
 
       if (!fieldProviders.isEmpty() || !methodProviders.isEmpty()) {
-        // [@Inject] Fields and methods in superclasses are injected before those in subclasses.
-        // we are walking up (getSuperclass), but spec says @Inject should be walking down the hierarchy
-        hierarchyBindings.addFirst(new ClassBindings(fieldProviders, methodProviders));
+        // Per JSR 330, fields and methods in superclasses are injected before those in subclasses.
+        // Since we are traversing upward in the hierarchy each injector is thus prepended to the
```

Comment:
"each injector is" doesn't make sense to me. There's something missing around there, it may be just commas.